### PR TITLE
chore(docs): update kafka docs and related ones

### PR DESCRIPTION
**Pull request by paologallinaharbur**
This PR:
 - fixes the link for the latest kafka installer pointing to the exe one
 - clarify the jmx page
 - removed in different places the requirement to restart the agent

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-install.mdx [hunk 1]
```diff
@@ -112,9 +111,7 @@ If you're not using the guided install, follow the instructions for your environ
     id="windows-install"
     title="Windows"
   >
-    1. Download the `nri-jmx` .MSI installer image from:
-
-       [https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-jmx/nri-jmx-amd64-installer.exe](https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-jmx/nri-jmx-amd64-installer.exe)
```
Comment (paologallinaharbur) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-install.mdx:117]: For docs folks: no idea if it is better to show the url when downloading something or not 😄 
I followed the same approach we had for kafka, but the old one might be better

**Comment by homelessbirds** [comment]
@paologallinaharbur Would you mind signing our CLA one more time? Thanks!

**Comment by paologallinaharbur** [comment]
@homelessbirds done!

**Comment by homelessbirds** [comment]
@bradleycamacho can you take a look at this one? Looks like it touches a lot of your work.

**Review by bradleycamacho — APPROVED** [review]

**Comment by bradleycamacho** [comment]
Hi @paologallinaharbur! Sorry, I didn't notice Sammie's notification earlier. Merging this in now.

**Pull request merged**